frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -0,0 +1,675 @@
+//! Dapps staking FRAME Pallet.
+
+use super::*;
+use frame_support::{
+    dispatch::{DispatchError, DispatchResult},
+    ensure,
+    pallet_prelude::*,
+    traits::{
+        Currency, CurrencyToVote, EnsureOrigin, EstimateNextNewSession, Get, LockIdentifier,
+        LockableCurrency, OnUnbalanced, UnixTime, WithdrawReasons,
+    },
+    weights::Weight,
+};
+use frame_system::{ensure_root, ensure_signed, offchain::SendTransactionTypes, pallet_prelude::*};
+use sp_runtime::{
+    traits::{CheckedSub, SaturatedConversion, StaticLookup, Zero},
+    Perbill, Percent,
+};
+use sp_std::{convert::From, prelude::*, result};
+
+mod impls;
+pub use impls::*;
+
+const STAKING_ID: LockIdentifier = *b"dapstake";
+
+#[frame_support::pallet]
+pub mod pallet {
+    use super::*;
+
+    #[pallet::pallet]
+    #[pallet::generate_store(pub(crate) trait Store)]
+
+    pub struct Pallet<T>(_);
+
+    #[pallet::config]
+    pub trait Config: frame_system::Config {
+        /// Maximum number of stakings per staker for dapps.
```

Comment:
Should this be declared as   #[pallet::constant] ?
The constant with the same name (save for the case) is declared on line 69.
So this should be removed I guess?